grouplogger.go

```diff
@@ -259,22 +259,25 @@ func (gl *GroupLogger) getMaxSeverity() logging.Severity {
 	return max
 }
 
+func NewUUID() string {
```

Comment:
Do you think this is generally useful to expose as a public function?

Reply:
Nope, good catch.